test/__init__.py

```diff
@@ -100,18 +100,17 @@ def wrapper(*args, **kwargs):
         msg = "Can't run {name} because the network is unreachable".format(
             name=test.__name__)
         try:
-            return test(*args, **kwargs)
+            sock = socket.create_connection((TARPIT_HOST, 80), 0.0001)
+        except socket.timeout:
+            pass
```

Comment:
If it times out creating a socket, why not use similar code to the other two exceptions? (I'm afraid that without a complimentary bug report explaining why, and without commit messages that explain *why* you'll be answering a lot of *why* questions in this review.)